src/mono/mono/metadata/object-internals.h

```diff
@@ -1658,15 +1658,7 @@ typedef enum {
 // Keep in sync with System.Runtime.Loader.AssemblyLoadContext
 typedef struct {
 	MonoObject object;
-	MonoObject *unload_lock;
-	MonoEvent *resolving_unmaned_dll;
-	MonoEvent *resolving;
-	MonoEvent *unloading;
-	MonoString *name;
 	MonoAssemblyLoadContext *native_assembly_load_context;
-	gint64 id;
-	gint32 internal_state;
-	MonoBoolean is_collectible;
```

Comment:
@CoffeeFlux are any of these fields going to be necessary for future collectible ALC work?